fdbserver/LogSystem.cpp

```diff
@@ -369,6 +374,54 @@ bool LogPushData::writeTransactionInfo(int location, uint32_t subseq) {
 	return true;
 }
 
+void LogPushData::addTLogGroups(const std::vector<TLogGroupRef>& groups, Version commitVersion) {
+	for (const auto& group : groups) {
+		pGroupMessageBuilders->emplace(group->id(),
+		                               std::make_shared<ptxn::ProxySubsequencedMessageSerializer>(commitVersion));
+	}
+}
+
+std::map<ptxn::TLogGroupID, ptxn::SerializedTeamData> LogPushData::getGroupMutations(
```

Comment:
do we need the ordering here in the returned map? looks like we can use an unordered_map

Reply:
the ordering is not needed. what's the advantage of using `unordered_map`?

Reply:
Inserting into the map is more efficient?